.github/actions/build_deb.sh

```diff
@@ -2,12 +2,28 @@
 
 echo "Building the SIL Kit packages"
 
-wget --no-proxy --no-check-certificate ${SILKIT_SOURCE_URL}/${SILKIT_VERSION}-${SILKIT_VERSION_SUFFIX}/libsilkit_${SILKIT_VERSION}.orig.tar.xz
-
-mkdir libsilkit-${SILKIT_VERSION}
+orig_tarball=${SILKIT_SOURCE_URL}/${SILKIT_VERSION}-${SILKIT_VERSION_SUFFIX}/libsilkit_${SILKIT_VERSION}.orig.tar.xz
```

Comment:
Should we switch to the offiicial source distrib package here? I.e., the Source subfolder from the distrib package.

Reply:
This would contain the vendored code from ThirdParty which is not needed/used in the orig. tarball. I think if we want to use source files from the original sil-kit repo, it might be best to do a non-recursive sparse clone of the git repo and then create the orig.tar.gz on the fly and include it in the artifacts.

Reply:
Sounds reasonable. Maybe do a follow-up PR later. (And maybe  just doing a `rm -rf ThirdParty` after unzipping would be enough😊)